assets/prefabs/Equipment/Gold/GoldBadgeOfLuck.prefab

```diff
@@ -0,0 +1,30 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:GoldBadgeOfLuck"
+    },
+    "DisplayName" : {
+        "name" : "Golden Badge For Luck",
+        "description" : "A Gold Badge that makes you luckier by an extent of 5!"
+    },
+    "EquipmentItem" : {
+        "level" : 6,
+        "quality" : 5,
+        "type" : "Badge",
+        "location" : "Badge",

```

Comment:
Can you change the location to `Ring` instead?
At the moment there are only some locations like head, arms (see the [player.prefab](https://github.com/Terasology/Equipment/blob/master/deltas/engine/prefabs/player/player.prefab) for details). If the item matches none of them, the stat change will not be applied. And adding a new slot for one item would be a bit too much (yet) :)